integration/docker/main_test.go

```diff
@@ -17,11 +19,33 @@ const (
 	shouldNotFail = false
 )
 
+var runtimeConfig RuntimeConfig
+
 func randomDockerName() string {
 	return RandID(30)
 }
 
 func TestIntegration(t *testing.T) {
+	var err error
+	runtimeConfigPath := DefaultRuntimeConfigPath
+
+	args := []string{"--kata-show-default-config-paths"}
+	cmd := NewCommand(Runtime, args...)
+	stdout, _, exitCode := cmd.Run()
+	if exitCode == 0 && stdout != "" {
+		for _, c := range strings.Split(stdout, "\n") {
+			if _, err = os.Stat(c); os.IsExist(err) {
```

Comment:
This doesn't look right to me - I don't think `IsExist()` does what we want here as `err` is expected to be `nil` when `os.Stat()` succeeds.